SX126X/SX126X_LoRaRadio.cpp

```diff
@@ -906,7 +906,7 @@ void SX126X_LoRaRadio::set_rx_config(radio_modems_t modem,
         }
 
         case MODEM_LORA: {
-            _rx_timeout_in_symbols = symb_timeout;
+            _rx_timeout_in_symbols = rx_continuous ? 0 : symb_timeout;
```

Comment:
FSK might require the same setting, so would it better to set it in the beginning of the method:

```
if (rx_continuous) {
    symb_timeout = 0;
}
```

Reply:
Good point. LoRaMac-node also sets it to zero for FSK. I'll push the change.